Snowflake.Data/Core/RestResponse.cs

```diff
@@ -78,6 +83,25 @@ internal class AuthnResponseData
         internal string ssoUrl { get; set; }
     }
 
+
+	internal class RenewSessionResponseData {
+
+		[JsonProperty(PropertyName = "sessionToken", NullValueHandling = NullValueHandling.Ignore)]
+		internal string sessionToken { get; set; }
+
+		[JsonProperty(PropertyName = "validityInSecondsST", NullValueHandling = NullValueHandling.Ignore)]
+		internal Int16 masterTokenValidityInSeconds { get; set; }

```

Comment:
This is actually validity in seconds for session token (ST)

Line 99 is validity in seconds for master token (MT)

Reply:
Oops, yeah I though that must have been the case. I just names things incorrectly. In case you didn't see I also incorrectly (or insufficiently) named validityInSecondsMT to validityInSeconds further down. I can change it on this branch for reference if you would like or you can just do it with your new branch. Thanks